parsers/myrddin.c

```diff
@@ -0,0 +1,46 @@
+/*
+ *   Copyright (c) 2016, Ori Bernstein
+ *
+ *   This source code is released for free distribution under the terms of the
+ *   GNU General Public License.
+ *
+ *   This module contains functions for generating tags for Myrddin language
+ *   files.
+ */
```

Comment:
Could you add a url or something for knowing myrddin language?
http://docs.ctags.io/en/latest/contributions.html#reference